src/client/app/graph/canvas/canvas.service.ts

```diff
@@ -123,6 +128,33 @@ export class CanvasService {
     }
   }
 
+  getCardById(id: string): Card {
+    // TODO(eyuelt): change CollaborativeList to CollaborativeMap
+    let cardsArray = this.cards.asArray();
```

Comment:
Seems like you have "let" here and "const" elsewhere. As a team, we should probably converge on what we like best.

Google's ES6 style guide says to use "const" everywhere possible, but I don't like how it semantically reads (versus, say, "let" and "var" from Swift lang). 

Which do you prefer? "let" or "const" by default?

Reply:
Good point. I prefer using "const" wherever possible, and using "let" whenever the variable isn't const. Same as the Google C++ style guide's "use const whenever possible" rule.

Yeah I think "let" was a poor choice of keyword given that it's used as "const" in other languages, but we're stuck with it since it is what we want semantics-wise.